README_DEV.md

```diff
@@ -1,16 +1,38 @@
-## Development
-### Android Studio
+# Development
+## Android Studio
 This project was developed using Android Studio Electric Eel, but you can also use Android Studio Dolphin since it's currently using Android Gradle Plugin version 7.3.1.
 
-### How to run the unit tests
+## Get up and running locally
+1. You need to install [the Java Developer Kit (JDK)](https://www.oracle.com/java/technologies/downloads/). The JDK is what powers Android Studio ability to build and run Android apps. Technically, our CI/CD builds Forage Android SDK using [JDK 11](https://github.com/teamforage/forage-android-sdk/blob/f2ff61ed27847b28d631f975c9a79e52f9258802/.github/workflows/CI.yaml#L21). However, as of this writing, the latest stable version is [JDK 17](https://www.oracle.com/java/technologies/downloads/#java17).
+    > ⚠️ NOTE: Installing the Java Developer Kit (JDK) is **not** the same thing as “[installing Java on your machine](https://www.oracle.com/ca-en/java/technologies/downloads/)”. The latter refers to installing the Java Runtime Environment (JRE), which is what you would install to run a Java application. As we are developing Java (Android) applications, we the JDK which includes (is a superset of) the JRE.
+2. Once you have installed the JDK of your choice to your local machine, you will need to make Android Studio aware by setting the `JAVA_HOME` environment variable to the path on your machine that the JDK resides. For example:
+   ```bash
+    # Windows
+   JAVA_HOME=C:\Program Files\Java\jdk-17
+   ```
+3. Confirm that the JDK and Android Studio are working as expected by running the [unit tests below](#How to run the unit tests). You should not see any errors. This may take a minute or two to run.
```

Comment:
this link is broken^

```suggestion
3. Confirm that the JDK and Android Studio are working as expected by running the [unit tests below](#how-to-run-the-unit-tests). You should not see any errors. This may take a minute or two to run.
```

Reply:
Nice catch! Addressed in [423105e](https://github.com/teamforage/forage-android-sdk/pull/44/commits/423105e670207ec61afd79cc00d4bd6e32d123e5)